src/collections/blog/2024/11-01-meet-the-maintainer-yash-sharma/post.mdx

```diff
@@ -0,0 +1,250 @@
+    helpful and always around to give a hand. You just don’t see that kind of
+    support in a lot of other open source projects.
+  </p>
+</div>
+
+<div class="interviewer">
+  <span>Hargun:</span>
+  <p>
+    <Link to="/projects">Layer5 projects</Link> has a number of active, open
+    source projects. You’ve been consistently contributing to a few of them.
+    Which one(s) are you currently focusing on?
+  </p>
+</div>
+
+<div class="interviewee">
+  <span>Yash:</span>
+  <p>
+    Absolutely, Layer5 has a number of active open source projects, and I’ve
+    contributed to quite a few of them. But my main focus has been on Meshkit,
+    Sistent, Meshery Extensions, and, of course, Meshery itself.
+  </p>
+</div>
+
+<div class="interviewer">
+  <span>Hargun:</span>
+  <p>What’s the coolest Meshery demo you have done/seen?</p>
+</div>
+
+<div class="interviewee">
+  <span>Yash:</span>
+  <p>
+    Haha good one, this year I got a chance to represent Meshery at KubeCon
+    which was really an amazing experience for me. I gave a demo of Kavas and
+    during representation and during that I was collaborating with another user
+    which really showed Meshery’s capability.
+  </p>
+</div>
+
+<div class="interviewer">
+  <span>Hargun:</span>
+  <p>What is your favorite feature in Meshery UI?</p>
+</div>
+
+<div class="interviewee">
+  <span>Yash:</span>
+  <p>
+    Meshery is an extensible platform and I like this feature of it, it gives
+    Meshery a whole new powerful capability to interact with various
+    technologies in the ecosystem and use it as an extension. My favourite one
+    is Kavas, though.
+  </p>
+</div>
+
+<div class="interviewer">
+  <span>Hargun:</span>
+  <p>What is your hot tip for working with Meshery that others may not know?</p>
+</div>
+
+<div class="interviewee">
+  <span>Yash:</span>
+  <p>
+    I definitely encourage you to join the meetings. They’re a great way to
+    connect with the maintainers in real time. Text chats just don’t do it
+    justice when it comes to letting them really get to know you. Video calls
+    are way better for that and give you a chance to show yourself to the
+    project leaders and maintainers.
+  </p>
+</div>
+
+<div class="interviewer">
+  <span>Hargun:</span>
+  <p>
+    Where do you see opportunities for contributors to get involved within
+    Meshery and Layer5 community?
+  </p>
+</div>
+
+<div class="interviewee">
+  <span>Yash:</span>
+  <p>
+    Meshery is growing very fast, it is the 9th fastest growing project under
+    CNCF. There are many opportunities are open for you to contribute. Start
+    with joining community and introducing yourself and sharing your skill. This
+    will help community managers to put you in right direction based on your
+    skills. All areas are consistently in need of help, including best at this
+    time to contribute like Meshery's CLI client, Meshery Server and digital
+    marketing.
```

Comment:
```suggestion
    Meshery is growing very fast, it is the 9th fastest growing project under
    CNCF. There are many opportunities are open for you to contribute. Start
    with joining community and introducing yourself and sharing your skill. This
    will help community managers to put you in right direction based on your
    skills. All areas are consistently in need of help, including best at this
    time to contribute like Meshery's CLI client, Meshery Server and digital
    marketing.
    
    Also out of 100+ repos that the Layer5 community stewards, two are currently closed: Kanvas and Cloud. Both of these projects were created and are actively being developed by open source contributors, though. There are ~15 open source contributors who working on it now, who meet daily. New contributors in the community routinely express interest and are invited to participate, and are extremely well supported as they do.

In other words, the source is currently closed (subject to change), but any community member here that shows interest, demonstrates competence and consistency of participation is readily invited to join in.
Please signal if this is you, and community managers will get you a tour of the code, introductions to other core team members, so that you might better assess your interest.
```